spark/src/test/scala/org/apache/comet/exec/CometColumnarShuffleSuite.scala

```diff
@@ -837,18 +759,13 @@ abstract class CometShuffleSuiteBase extends CometTestBase with AdaptiveSparkPla
 
   test("fix: Native Unsafe decimal accessors return incorrect results") {
     Seq(10, 201).foreach { numPartitions =>
-      withSQLConf(
-        CometConf.COMET_EXEC_ENABLED.key -> "true",
-        CometConf.COMET_COLUMNAR_SHUFFLE_ENABLED.key -> "true",
-        CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true") {
```

Comment:
Hmm, these configs are used to produce a specific situation. If they are changed, you won't fail the test but maybe it just tests nothing.

Reply:
If you move the common configs and it should be good. There are many tests so we should be careful if no one is missed or with changed configs.